packages/bundler/src/modules/initServer.ts

```diff
@@ -2,13 +2,14 @@ import { ExecutionManager } from './ExecutionManager'
 import { BundlerReputationParams, ReputationManager } from './ReputationManager'
 import { MempoolManager } from './MempoolManager'
 import { BundleManager } from './BundleManager'
-import { ValidationManager } from './ValidationManager'
+import { ValidationManager } from '@account-abstraction/validation-manager'
 import { EntryPoint__factory } from '@account-abstraction/contracts'
 import { parseEther } from 'ethers/lib/utils'
 import { Signer } from 'ethers'
 import { BundlerConfig } from '../BundlerConfig'
 import { EventsManager } from './EventsManager'
 import { getNetworkProvider } from '../Config'
+import { abi } from './Types'
```

Comment:
this abi is a set of ABIs used to simplify decoding of call stacks. it is not a global parameter.